service/raft/group0_state_machine.hh

```diff
@@ -122,4 +123,8 @@ public:
     future<> abort() override;
 };
 
+bool should_flush_system_topology_after_applying(const mutation& mut, const data_dictionary::database db);
+
+future<> write_mutations_to_database(storage_proxy& proxy, gms::inet_address from, std::vector<canonical_mutation> cms);
```

Comment:
The function name is pretty generic. However, its implementation is specific to only certain types of mutations (mutations with specific content). The name and preferably a comment should make that clear